takwimu/templates/takwimu/_includes/navbar/search.html

```diff
@@ -0,0 +1,50 @@
+<div class="modal dropdown-menu-lg text-light container-fluid shadow"
+    id="search-modal" tabindex="-1" role="dialog" style="top: 72px; height: 400px;">
+    <button type="button" class="close position-absolute" style="top: 1em; right: 1em;" data-dismiss="modal" aria-label="Close">
+        <span aria-hidden="true">&times;</span>
+    </button>
+    <div class="container" role="document" style="margin-top: 150px;">
+        <form>
+            <div class="input-group col-6 mx-auto">
+                <input type="text" class="form-control border-0 py-3 px-5 text-center" id="search-takwimu" placeholder="Search Takwimu">
+
+                <button type="submit" class="btn btn-search">
+                    <i class="fa fa-search"></i>
+                </button>
+            </div>
+        </form>
+    </div>
+</div>
+<style>
+    .close {
+        font-size: 4rem; 
+        font-weight: 100; 
+        color: #fff; 
+        text-shadow: none; 
+        line-height: 0;
+        opacity: 1;
+    }
+    .btn-search {
+        position: absolute;
+        right: 1em;
+        bottom: 1em;
+        color: #fff;
+        background: transparent;
+    } 
+    input[type="text"]#search-takwimu {
+        font-size: 35px;
+        font-weight: 300;
+        background: transparent;
+        border-bottom: solid 2px #ffffff!important;
+        color: #fff;
+    }
+    input[type="text"]#search-takwimu:focus {
+        box-shadow: none;
+        outline: 0 none;
+    }
+    #search-takwimu::-webkit-input-placeholder, #search-takwimu:-moz-placeholder  {
+        color: rgba(255, 255, 255, 0.45);
+        font-size: 35px;
+        font-weight: 300;
+    }
+</style>
```

Comment:
Let's move all these styles to `_index.html` and use `.dropdown-menu-lg` selector instead.

Reply:
@DavidLemayian 
I have moved the styles to the `_index.html` file and kept the popover functionality,

Let me know if it needs more touch